examples/kinfu/README.md

```diff
@@ -0,0 +1,34 @@
+# Azure Kinect - OpenCV KinectFusion Example
+
+## Introduction
+
+The Azure Kinect - OpenCV KinectFusion example shows how to use the Azure Kinect SDK and device with the KinectFusion example from opencv_contrib's rgbd module (https://github.com/opencv/opencv_contrib/tree/master/modules/rgbd). The example demonstrates how to feed calibration and depth images from Azure Kinect APIs to the OpenCV KinectFusion module with undistortion. We render live KinectFusion results in this example as well as generate fused point cloud as a ply file when quitting application.
```

Comment:
This is a very long paragraph to have on one line. For eadability, can you break this up into multiple lines (of a certain col limit).